test-cases/longevity/longevity-twcs-48h.yaml

```diff
@@ -21,3 +21,5 @@ nemesis_during_prepare: false
 space_node_threshold: 64424
 
 user_prefix: 'longevity-twcs-48h'
+
+post_prepare_cql_cmds: "ALTER TABLE scylla_bench.test with gc_grace_seconds = 300 and default_time_to_live = 21600 and compaction = {'class':'TimeWindowCompactionStrategy', 'compaction_window_size': 60, 'compaction_window_unit': 'MINUTES'};"
```

Comment:
But we don't have prepare_workload here.
What would happen in this case? Do we even have the table?
If not, you need to prepare schema or run a prepare workload that just create the table with 1 partition or something like that.

Reply:
TWCS tests are run with `longevity_twcs_test.py`. This test define the prepare command in code explicitly. you are right, it could be confusing. may be it is better to move the setting also to code inside method of test?

Reply:
I don't mind it'll be here, as long as it's really running :)